compose/dashboard_conf/nginx.conf

```diff
@@ -16,7 +16,8 @@ server {
     }
 
     location /apisix/admin {
-        proxy_pass http://manager:8080/apisix/admin;
+        proxy_pass http://192.17.5.15:8080/apisix/admin;
```

Comment:
I don't think it's a good idea to use a fixed IP here, @gxthrj @bzp2010 what do you think?

Reply:
It should not be changed here, not a problem here